src/importlinter/domain/helpers.py

```diff
@@ -34,6 +34,33 @@ def pop_imports(
     return removed_imports
 
 
+def import_expressions_to_imports(
+    graph: ImportGraph, expressions: Iterable[ImportExpression]
+) -> List[Dict[str, Union[str, int]]]:
+    imports: List[DirectImport] = []
+    for expression in expressions:
+        was_any_removed = False
```

Comment:
This variable name no longer makes sense in this context, maybe we should call it `matched` or something like that?

Reply:
done